test/python/visualization/test_circuit_drawer.py

```diff
@@ -141,74 +141,6 @@ def test_latex_output_file_correct_format(self):
                     self.assertIn(im.format.lower(), filename.rsplit(".", maxsplit=1)[-1])
             os.remove(filename)
 
-    def test_wire_order(self):
```

Comment:
Still a valid test, I think?

Reply:
Restored in: https://github.com/Qiskit/qiskit/pull/13506/commits/b47f82c00ca9c384f04be7d9655502d7e666e1a1